app/services/train_api.rb

```diff
@@ -0,0 +1,12 @@
+class TrainApi
+  include HTTParty
+
+  def initialize
+    self.class.base_uri 'localhost:3001'
```

Comment:
we still gotta make sure we arent committing the url part here as hardcoded, should spend some time setting up dotenv